xinference/model/llm/pytorch/core.py

```diff
@@ -215,7 +216,10 @@ def generate(
         def generator_wrapper(
             prompt: str, device: str, generate_config: PytorchGenerateConfig
         ) -> Iterator[CompletionChunk]:
-            if "falcon" in self.model_family.model_name:
```

Comment:
`"falcon" in self.model_family.model_name:` already include `falcon` and `falcon-instruct`.
No need to change this command.